### ISSUE TITLE
Could not load image

### ISSUE THREAD

**Issue by daretmavi** [bell07/minetest-smart_inventory#6]
If I open smart inventory and click creative, I get en error in console:
generateImage(): Could not load image "right_arrow.png" while building texture; Creating a dummy image

This is from debug.txt:
```
-------------
  Separator
-------------

2019-12-29 21:33:23: ERROR[Main]: Irrlicht: Invalid size of image for OpenGL Texture.
2019-12-29 21:33:23: WARNING[Main]: WARNING: minetest.setting_* functions are deprecated.  Use methods on the minetest.settings object. (at ...st/mods/smart_inventory-master/libs/simple_po_reader.lua:1)
2019-12-29 21:33:23: WARNING[Main]: WARNING: minetest.setting_* functions are deprecated.  Use methods on the minetest.settings object. (at ...st/mods/smart_inventory-master/libs/simple_po_reader.lua:8)
2019-12-29 21:33:23: WARNING[Main]: WARNING: minetest.setting_* functions are deprecated.  Use methods on the minetest.settings object. (at ...et/.minetest/mods/smart_inventory-master/libs/filter.lua:209)
2019-12-29 21:33:23: WARNING[Main]: WARNING: minetest.setting_* functions are deprecated.  Use methods on the minetest.settings object. (at ....minetest/mods/smart_inventory-master/pages/creative.lua:1)
2019-12-29 21:33:23: ACTION[Main]: World at [/home/user/.minetest/worlds/test_smart]
2019-12-29 21:33:23: ACTION[Main]: Server for gameid="minetest" listening on 0.0.0.0:30000.
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_cyan for result dye:cyan
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_pink for result dye:pink
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_dark_grey for result dye:dark_grey
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_magenta for result dye:magenta
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_brown for result dye:brown
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_grey for result dye:grey
2019-12-29 21:33:23: [Server]: [smartfs_inventory] no items matches group: group:flower,color_dark_green for result dye:dark_green
2019-12-29 21:33:26: WARNING[Main]: Irrlicht: PNG warning: iCCP: known incorrect sRGB profile
2019-12-29 21:33:26: WARNING[Main]: Irrlicht: PNG warning: iCCP: known incorrect sRGB profile
2019-12-29 21:33:26: WARNING[Main]: Irrlicht: PNG warning: iCCP: known incorrect sRGB profile
2019-12-29 21:33:27: ACTION[Server]: Giving initial stuff to player Username
2019-12-29 21:33:27: ACTION[Server]: Username [127.0.0.1] joins game. 
2019-12-29 21:33:27: ACTION[Server]: Username joins game. List of players: Username
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to get_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:731)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to get_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:420)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to set_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:420)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to get_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:464)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to set_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:464)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to get_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:464)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to set_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/crafting.lua:464)
2019-12-29 21:33:27: WARNING[Server]: Deprecated call to get_attribute, use MetaDataRef methods instead. (at ....minetest/mods/smart_inventory-master/pages/creative.lua:277)
2019-12-29 21:33:32: ERROR[Main]: generateImage(): Could not load image "right_arrow.png" while building texture; Creating a dummy image
2019-12-29 21:33:52: ERROR[Main]: Irrlicht: Invalid size of image for OpenGL Texture.
2019-12-29 21:33:52: ACTION[Server]: Username leaves game. List of players: 
2019-12-29 21:33:52: ACTION[Main]: Server: Shutting down

```

**Comment by bell07** [comment]
The mod did used right_arrow.png and left_arrow.png from "default" mod (till 0.4.17). Now the both pictures are away from Minetest Game / from default mod, so new textures required. Requested in Forum, but if anyone provides nice arrow textures I can apply them to the mod. Note, the textures needs to give place for the number. The affected buttons are "Page back/ Page Next" in creative items list.

**Comment by daretmavi** [comment]
![left_arrow](https://user-images.githubusercontent.com/47914240/71785452-f9a92800-2fff-11ea-93b6-f40ab42fb590.png)
![right_arrow](https://user-images.githubusercontent.com/47914240/71785453-f9a92800-2fff-11ea-8732-7aefc22cde1d.png)

If you want, you can use these ones.
I created them for myself, you are free to use them.

**Comment by bell07** [comment]
@daretmavi, thank you! I added the buttons to the mod. Which license do the textures have? 
I used the same as other textures (WTFPL) but let me know it is wrong

**Comment by bell07** [comment]
Fixed in 8ff56a3322f48a2f2b2d298fc794d6ff65e627a1
